dev/puppet-server.conf.sample

```diff
@@ -115,3 +115,8 @@ authorization: {
         }
     ]
 }
+
+# general puppetserver settings
+puppetserver: {
+    environment-class-cache-enabled: true
+}
```

Comment:
Ah, okay - didn't see @KevinCorcoran's comment before I made the above comment. If we do already have settings in this section, then I'm cool with this I guess. Might be worth adding those settings commented out to the example config file.